PXWeb/HttpResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace PxWeb
+{
+    public class HttpResponse
```

Comment:
Should rename the class so that it is not confused the the HttpResponse class in System.Web. Maybe something like ChachedResponse